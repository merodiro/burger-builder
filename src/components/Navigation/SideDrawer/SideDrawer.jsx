import React, { Fragment } from 'react'

import classes from './SideDrawer.module.css'
import Logo from '../../Logo/Logo'
import NavigationItems from '../NavigationItems/NavigationItems'
import Backdrop from '../../UI/Backdrop/Backdrop'

const SideDrawer = ({ closed, open }) => {
  let attatchedClassed = [classes.SideDrawer, classes.Close]
  if (open) {
    attatchedClassed = [classes.SideDrawer, classes.Open]
  }
  return (
    <Fragment>
      <Backdrop clicked={closed} show={open} />
      <div className={attatchedClassed.join(' ')}>
        <div className={classes.Logo}>
          <Logo />
        </div>
        <nav>
          <NavigationItems />
        </nav>
      </div>
    </Fragment>
  )
}

export default SideDrawer
