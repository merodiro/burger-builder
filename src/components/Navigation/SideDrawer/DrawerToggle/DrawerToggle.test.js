import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'

import SideDrawer from './DrawerToggle'

describe('<DrawerToggle>', () => {
  it('renders the same output', () => {
    const component = shallow(<SideDrawer />)
    expect(component).toMatchSnapshot()
  })
})
