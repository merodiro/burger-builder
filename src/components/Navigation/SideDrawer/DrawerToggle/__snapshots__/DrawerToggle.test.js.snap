// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DrawerToggle> renders the same output 1`] = `
<div>
  MENU
</div>
`;
